src/material/sidenav/sidenav.md

```diff
@@ -209,6 +209,11 @@ Similarly, the `<mat-sidenav-content>` should be given a role based on what it c
 represents the primary content of the page, it may make sense to mark it `role="main"`. If no more
 specific role makes sense, `role="region"` is again a good fallback.
 
+#### Focus management
+To enable auto focus within `<mat-drawer>` component, you need to set `[autoFocus]=true`. 
```

Comment:
I'm not sure we need to mention this since it's turned on by default.

Reply:
/**
   * Whether the drawer should focus the first focusable element automatically when opened.
   * Defaults to false in when `mode` is set to `side`, otherwise defaults to `true`. If explicitly
   * enabled, focus will be moved into the sidenav in `side` mode as well.
   */
this is the comment on autoFocus property, so I thought it would be better to mention it. maybe I could explicitly tell that autoFocus is default in side mode?

Reply:
Ah, I forgot that it depends on the mode. Then let's keep it, but we should probably reword a bit. E.g. mention that by default we capture and restore focus depending on the mode and that it can be overridden using the `autoFocus` input.

Reply:
"Auto Focus within `<mat-drawer>` component is enabled by default, if it's <b>not</b> in 'side' mode. We can manage it explicitly, using `autoFocus` input property."